pom.xml

```diff
@@ -67,6 +67,7 @@
         <version.pegasus>27.7.18</version.pegasus>
         <version.smile>2.6.0</version.smile>
         <version.weld>3.1.4.Final</version.weld>
+        <version.cucumber.reporting.plugin>5.7.5</version.cucumber.reporting.plugin>
```

Comment:
Q: Why here instead of the `build-parent`?

Reply:
We use the value in our enforcer extension module, which extends from the root pom instead of build-parent. (That's required to avoid circular dependency, as build-parent uses the enforcer extension itself). Prior to this change, we had the version specified both in the enforcer pom and in build-parent.  This consolidates them into one place.